### PR TITLE
Added Port 443 to Capsule incoming traffic table

### PR DESCRIPTION
**Pull request by bangelic**
Per SAT-6405, I added port 443 in the Destination Port of the Capsule incoming traffic table where port 8443 is mentioned.

Cherry-pick into:

* [x] Foreman 3.3
* [x] Foreman 3.2
* [x] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

**Comment by evgeni** [comment]
If you install a fresh Katello/Satellite/Orcharino, you won't need port 8443 - all newly deployed systems will use 443 exclusively.
However, if you have client hosts that have been upgraded since before Foreman 3.1, those will be still talking via 8443.

Thus it would be nice if we could somehow point out that 8443 is legacy/deprecated and only really needed in existing setups.

I *think* this is done best by having two lines, one for 443 with the current text, and one for 8443 with some additional sentence explaining the situation.

**Comment by bangelic** [comment]
@evgeni Let me now if that sentence makes sense for Port 8443.

**Comment by maximiliankolb** [comment]
Is this related to #1344?

**Comment by evgeni** [comment]
> Is this related to #1344?

It is related as in "the same code change triggered both updates to the docs", yes.

File: guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc
Comment (evgeni) [inline guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc:55]: I'm torn on the wording, also partially because I'm unhappy with the *old* description (which you didn't write).

The purpose of the port in this scenario is to allow the client to talk to the RHSM API for obtaining and updating subscription data. This happens during *registration* but also afterwards (subscription-manager regularly updates subscriptions of the content host).

Now, with the changes we implemented, new hosts will register via `443` and store that in their configs, so that later communication also happens via `443`. However, existing hosts (those, that are already registered) don't get a configuration update and continue talking via `8443`.

Thus saying it's used for *registration* seems odd to me -- those hosts aren't doing any registration anymore as they are already registered. They do, however, keep on using that port for communication until reconfigured by the admin.

I wouldn't block this PR on improving this wording, but that improvement should happen!

Comment (bangelic) [reply guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc:55]: @evgeni If you have something specific you want me to change from "Content Host registration", let me know, otherwise we can keep as is and address in a future PR and we can merge this one. Up to you.

**Review by evgeni — APPROVED** [review]
I'm good with the change (as it documents the current state of deployments).

We should def improve the wording in a later PR tho

File: guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc
Comment (bangelic) [reply guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc:55]: @evgeni All I could think of was something related to "Content Host communication" but I'm not a subject matter expert so I defer to all of you who know this stuff better than me. :-)

Comment (ehelms) [reply guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc:55]: No idea, so :roller_skate: on

**Comment by bangelic** [comment]
@evgeni Sounds good. One of the checks was unsuccessful. Do I need to push this again before it gets merged?

**Comment by evgeni** [comment]
You can rebase and the red check will go away.

**Comment by bangelic** [comment]
> You can rebase and the red check will go away.

@evgeni Done!

**Comment by bangelic** [comment]
@evgeni Is this good to be merged?

**Comment by bangelic** [comment]
> @bangelic Please rebase to master and squash your commits. Reword your commit message according to https://cbea.ms/git-commit/#seven-rules as stated in the contributing guidelines and please avoid downstream terminology (at least in the subject).

@maximiliankolb Done!

**Comment by maximiliankolb** [comment]
@bangelic It's still past tense and says "capsule" in the commit message subject line.

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   96920db07..a3df0205b  3.3 -> 3.3
   2b1b47f21..09b0c000c  3.2 -> 3.2
   dd984e2d5..7b5b77bef  3.1 -> 3.1

**Pull request merged**
